packages/csvviewer/src/model.ts

```diff
@@ -83,9 +83,9 @@ export class DSVModel extends DataModel implements IDisposable {
     this._parseAsync();
 
     // Cache the header row.
-    if (header === true && this._columnCount > 0) {
+    if (header === true && this._columnCount! > 0) {
```

Comment:
It was really hard to track down when the various variables in the CSV viewer code was null/undefined, so for now I simply added non-null asserts everywhere. This commit has a lot of other such asserts as well, and basically maintains the status-quo, even if they might have revealed some incorrect nullable handling. The main point to review this for is if any fix of an incorrect call here would cause an API breakage (we could probably put those as "fixing" typings as a bug, so might be ok).